src/components/widgets/sales-leads-funnel/sales-leads-funnel.tmpl.html

```diff
@@ -4,7 +4,7 @@
     <h4>Widget settings</h4>
 
     <div setting-organizations parent-widget="widget" class="part" deferred="::orgDeferred" />
-    <div setting-params-picker options="statusOptions" param="status_selection" parent-widget="widget" class="part" deferred="::paramsPickerDeferred" />
+    <div setting-params-picker options="statusOptions" param="leads_per_status" has-reach="hasReach" parent-widget="widget" class="part" deferred="::paramsPickerDeferred" />
```

Comment:
Same thing here, I don't see why the parameter naming is changed...
A metadata is an information about the widget's configuration that will be sent to the back-end. If you change its naming it can potentially break the back-end as it won't receive the parameter it's expecting.

Plus, what about the widgets that are already in production and use the original configuration param name?